taxi/urls.py

```diff
@@ -13,6 +12,10 @@
     ManufacturerCreateView,
     ManufacturerUpdateView,
     ManufacturerDeleteView,
+    DriverCreateView,
+    DriverDeleteView,
+    LicenseNumberUpdateView,
+    AssignDriverView,
```

Comment:
it's not an error but I would recommend getting used to sorted imports, maps etc., improves readability, eases maintenance, tools like `isort` can help